public/scripts/data-util.js

```diff
@@ -1,17 +1,19 @@
 var DataUtil = (function() {
-    var data = {};
-
-    var cluster_name_mapping = null;
+  var data = {};
+
+  var cluster_name_mapping = [];
+  var job_name_prefix = '';
+
+  $.ajax({
+    async: false,
```

Comment:
If it's not too crazy I think it would be worthwhile now that we're starting to use the config more on the client.